### ISSUE TITLE
Missing accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#15]
Hello! I noticed that the study group map is missing some accessibility features. It would be great if you could add them to make the map more inclusive and usable for everyone. Thank you for your hard work!
